server/server.js

```diff
@@ -0,0 +1,41 @@
+const express = require('express')
+const app = express();
+const mongoose = require('mongoose')
+const Game = require("./gameModal/gameModal")
+app.use(express.json())
+const cors = require('cors');
+
+app.use(cors());
+
+mongoose.set("strictQuery",false);
+
+mongoose.connect('mongodb://127.0.0.1:27017/gamedex').then(() => {
```

Comment:
Add `mongodb://127.0.0.1:27017/gamedex` to `.env`, later you need to migrate to Atlas when you deploy your application.